js/models/TimeTrigger.js

```diff
@@ -4,8 +4,8 @@ TimeTrigger.schema = {
   primaryKey: 'id',
   properties: {
   	id: 'string',
-  	formId: 'string',
-  	surveyId: 'string',
+  	formId:  {type: 'string', default: 'none'},
+  	surveyId:  {type: 'string', default: 'none'},
```

Comment:
From what I found Realm can perform automatic migrations, having the default set can prevent a few crashes when updating objects when calling these new properties.

There's no easy way to check if a value is null with a query, as it forces strict type casting.